src/components/Header/__snapshots__/Header.test.js.snap

```diff
@@ -9,9 +9,14 @@ exports[`Header match snapshots 1`] = `
   <div
     className="wkp-header-right"
   >
-    <UNDEFINED
-      appBaseUrl="http://foo.de"
-    />
+    <div
+      className="wkp-header-right-bloc"
+    >
```

Comment:
Remove this div and creates a component for the Logo. So the alignment will be done properly by flex.